include/soci/column-info.h

```diff
@@ -11,7 +11,6 @@
 #include "soci/soci-backend.h"
 #include "soci/type-conversion.h"
 #include "soci/values.h"
-
```

Comment:
Minor, but it's better to try to avoid making whitespace-only changes in the commits with significant changes.